src/components/DisplayTaxonName.js

```diff
@@ -8,18 +8,26 @@ import { useTranslation } from "react-i18next";
 import { generateTaxonPieces } from "sharedHelpers/taxon";
 
 type Props = {
-  item: Object,
+  observation: Object,
```

Comment:
Hah, this is like the opposite of my previous feedback: `DisplayTaxonName` is something we're going to use all over the place, not just in the context of an observation, so it would be more appropriate for it to accept a `taxon` prop and a `user` prop, since those will always be relevant to displaying a taxon name.

Reply:
No worries lol, small change. Will fix that.